pkg/feature/builder.go

```diff
@@ -32,6 +32,11 @@ func (fb *featureBuilder) For(spec *v1.DSCInitializationSpec) *featureBuilder {
 			ServiceMeshSpec: &spec.ServiceMesh,
 			Serving:         &infrav1.ServingSpec{},
 			AppNamespace:    spec.ApplicationsNamespace,
+			AuthorinoConfigs: AuthorinoConfigs{
+				CrName:         "authorino-mesh-authz-provider",
+				ContainerImage: "quay.io/kuadrant/authorino:v0.16.0",
```

Comment:
Isn't it better to let the authorino operator use whatever is the default image?

Reply:
Agree, let's remove it. It shouldn't be hardcoded like this. We will need to find a clean way of changing it, but this can be a part of a parallel work regarding installation of the Authorino controller itself. /cc @cam-garrison